ci/packages/package.go

```diff
@@ -222,7 +222,7 @@ func checkVend() error {
 		fmt.Println("Tool 'vend' already installed")
 		return nil
 	}
-	err := goGet("github.com/nomad-software/vend")
+	err := goGet("github.com/mysteriumnetwork/vend")
```

Comment:
We need a patched vendor tool to vendor our patched wireguard. It feels like we doing something wrong. :D

Reply:
This is only because gomobile does not support gomodules yet

Reply:
Let's patch gomobile to support gomodules. 💃

Reply:
We could contribute but that will take a while.